### ISSUE TITLE
Mark PrintDescriptionsAsComments as obsolete for v4?

### ISSUE THREAD

**Issue by sungam3r** [graphql-dotnet/graphql-dotnet#2762]
And remove in v5?

**Comment by Shane32** [comment]
@sungam3r Is this done?

**Comment by sungam3r** [comment]
No. `SchemaPrinterOptions.PrintDescriptionsAsComments` property was not marked as obsolete and I would not bother with it since I want to completely change printer. `SchemaPrinterOptions` with high probability will become useless. GraphQL-Parser implements all that stuff now or allow to implement it in easy way (like `IncludeDeprecationReasons`). So I'm OK to close this on (as well as #2887) with resolution "will be fixed as part of #2868".

**Comment by sungam3r** [comment]
HA! See https://github.com/graphql-dotnet/graphql-dotnet/issues/2887#issuecomment-1023763496 and your answer.

**Comment by sungam3r** [comment]
Let's move gradually to the full rewrite.
